code/src/index.css

```diff
@@ -7,7 +7,124 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+* {
+  box-sizing: border-box;
+}
+
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+main {
+  max-width: 1200px;
+  margin: 0 auto;
+  width: 100%;
+  padding: 0 50px;
+}
+
+/*MovieList styling*/
+
+.movie-list {
+  display: flex;
+  flex-wrap:wrap
+}
+
+.movie-list a {
+  width: 25%;
+  text-decoration: none;
+  color: white;
+  position: relative;
+}
+
+.movie-list a img {
+  width: 100%;
+}
+
+.movie-list .movie-card-details {
```

Comment:
I don't understand why you write two class names after each other. Do you style both at the same time? I think you're only adressing the second one, but then you don't need another class before it.